features/theme_gem.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Building Theme Gems
+  As a hacker who likes to share my expertise
+  I want to be able to make a bonafide rubygem off my theme
+  In order to share my awesome style skillz with other Jekyllites
+
+  Scenario: Generating a new Jekyll Theme
+    When I run jekyll new-theme my-cool-theme
+    Then I should get a zero exit status
+    And the my-cool-theme directory should exist
+
+  Scenario: Checking if a bonafide Theme gem will be built from generated scaffolding
+    When I run jekyll new-theme my-cool-theme
+    Then I should get a zero exit status
+    And the my-cool-theme directory should exist
+    When I decide to build the theme gem
+    Then I should get a zero exit status
+    When I run git add .
+    Then I should get a zero exit status
```

Comment:
These check the exit status of the Jekyll build, I think. They'll have to be updated to see what the latest command was and read its build status.